src/views/Team.vue

```diff
@@ -76,7 +76,6 @@ export default defineComponent({
                     image: '/assets/images/pics/axtloss.jpeg',
                     links: {
                         githubUsername: 'axtloss',
-                        twitterUsername: 'brombinmirko',
                         mastodonLink: 'https://blahaj.zone/@xenia',
```

Comment:
this isnt accurate anymore either, the current one is `https://fosstodon.org/@axtlos`

Reply:
Thanks! I've updated the link